src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -375,6 +375,11 @@ public bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out CustomNodeWork
             return false;
         }
 
+        public bool TryGetFunctionWorkspace(Guid id, bool isTestMode, out ICustomNodeWorkspaceModel ws)
+        {
+            return TryGetFunctionWorkspace(id, isTestMode, out ws);
+        }
```

Comment:
I assume that here I had to return bool as it was before, when `TryGetFunctionWorkspace` took `CustomNodeWorkspaceModel`. 

But if I rewrite `TryGetFunctionWorkspace` and it'll take `ICustomNodeWorkspaceModel`, this will cause huge number of changes! For now I decided leave `TryGetFunctionWorkspace` without changes and just implement new interface.